src/sockperf.cpp

```diff
@@ -2208,6 +2208,13 @@ static int parse_common_opt(const AOPT_OBJECT *common_obj) {
 #endif /* DEFINED_TLS */
     }
 
+#if defined(__arm__) && !defined(__aarch64__)
+    if (!s_user_params.b_no_rdtsc) {
+        log_msg("AArch32 target build does not support rdtsc, use --no-rdtsc");
+        rc = SOCKPERF_ERR_BAD_ARGUMENT;
+    }
+#endif
```

Comment:
Please instead of this check extend following configurable
https://github.com/Mellanox/sockperf/blob/sockperf_v2/src/sockperf.cpp#L2106

Something as
```
#if !defined(__windows__) && !(defined(__arm__) && !defined(__aarch64__))
#endif
```

Reply:
> So instead of creating an error, the `b_no_rdtsc` value should be set by default for AArch32 targets?
> 
yep, just set as default value and remove usage of `--no-rdtsc`
I missed this place https://github.com/Mellanox/sockperf/blob/sockperf_v2/src/sockperf.cpp#L269

> Then I would have to alter the default value in https://github.com/Mellanox/sockperf/blob/sockperf_v2/src/defs.h#L740 , as skipping the argument parsing for AArch32 keeps the wrong value (using rdtsc)